app/controllers/application_controller.rb

```diff
@@ -1380,7 +1380,13 @@ def find_checked_items_with_rbac(klass, prefix = nil)
     items
   end
 
-  # Common Saved Reports button handler routines
+  # Test RBAC in case there is only one record, not checked by checkbox
+  def test_item_with_rbac(klass, id)
```

Comment:
added in https://github.com/ManageIQ/manageiq-ui-classic/pull/779/commits/ec883d5066872472ecd17c65a760b52d377b6a6f